brewtils/plugin.py

```diff
@@ -843,94 +841,202 @@ def _validate_system(self):
     # These are provided for backward-compatibility
     @property
     def bg_host(self):
+        """
+        .. deprecated:: 3.0
+           bg_host is now in ``_config`` (``plugin._config.bg_host``)
+
+        Provided for backward-comptibility
```

Comment:
Just noticing that I didn't make any assertions about when this will be removed.  I'm open to going back and claiming it'll go away in 4.0.  I don't feel too strongly about it.

Reply:
Yeah, I don't think we need to be too worried about it. I'm guessing nobody uses these anyway, I added them so 3.0 didn't break things on the off-change somebody was.